apps/web-client/app.js

```diff
@@ -34,13 +35,16 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.use((req, res, next) => {
   req.deps = {
     request,
+    url,
     photoApiUrl: `http://localhost:${process.env.API_PORT}`,
     s3Bucket: process.env.S3_BUCKET,
   };
   next();
 });
 
 // Routes
-app.use('/', require('./middleware/homepage'));
+app.get('/', require('./middleware/homepage'));
+
+app.post('/photo', require('./middleware/upload'));
```

Comment:
Ah you're right. Even the scaffolding puts them like that. In my old place, our middleware is where we put our `(req, res, next) => {...}` handlers. Just confused it. But it's not really applicable here.

In short, I'm changing these to routes